Gemfile

```diff
@@ -8,7 +8,7 @@ gem "rake"
 
 gem "kubeclient", :git => "https://github.com/abonas/kubeclient", :branch => "master"
 gem "manageiq-messaging", "~> 0.1.2"
-gem 'topological_inventory-api-client',         :git => "https://github.com/eclarizio/topological_inventory-api-client", :branch => "master"
+gem 'topological_inventory-api-client',         :git => "https://github.com/ManageIQ/topological_inventory-api-client", :branch => "master"
```

Comment:
I am going to rename the gem to https://github.com/ManageIQ/topological_inventory-api-client-ruby, so you'll hae to change the git part.